src/swagger-specs/flow-service.yaml

```diff
@@ -35,26 +35,27 @@ host: platform.adobe.io
 basePath: /data/foundation/flowservice
 tags:
   - name: Connections
-    description: Connections retain information regarding how to connect to a source or target. This includes a unique connection ID, which is necessary in order to explore or navigate files and objects.
+    description: Base connections retain information between your source and Adobe Experience Platform. They include metadata, a partial view of the credentials used to authenticate a given source, and the unique base connection ID that corresponds to the queried connection. The base connection ID represents your authenticated connection and allows you to explore and inspect the type, structure, and contents of the data that you want to bring to Experience Platform. The base connection ID is required to create a source connection. An instance of a base connection is specific to an organization.
```

Comment:
Would this still mean the same if I changed the order of the two sentences?

"The base connection ID is required to create a source connection. It also represents your authenticated connection and allows you to explore and inspect the type, structure, and contents of the data that you want to bring to Experience Platform. "